assets/css/virtual_sign.css

```diff
@@ -10,5 +10,5 @@
   height: 40px;
   padding-top: 8px;
   white-space: pre;
-  width: 24ch;
+  width: 25ch;
```

Comment:
Did we make a mistake here originally and signs do in fact accommodate 25 characters?

Reply:
In a sense... I suppose. I wanted to give a little "breathing room" to the right of the clock, since 24 characters is exactly the amount to the end of the clock. Since this is the `width`, I couldn't apply padding on that side (it eats into the line), and margin doesn't have the background color. I could rework it, but I thought just adding one more character of space was simple enough.

Reply:
> Since this is the `width`, I couldn't apply padding on that side (it eats into the line)

It shouldn't work that way... is this element ending up with a non-standard `box-sizing`?

> By default in the CSS box model, the `width` and `height` you assign to an element is applied only to the element's content box. If the element has any border or padding, this is then added to the `width` and `height` to arrive at the size of the box that's rendered on the screen. [(source)](https://devdocs.io/css/box-sizing)

Reply:
Aha! Okay, good, I'm not crazy. I was going to respond to this comment, "I just did it this way without thinking much, but now realize I should use padding instead", but then when I added padding... it didn't work. I had a crisis of faith about my understanding of padding. So in that case, I think it must be overconstrained by a parent's width or some combination of flex shrink/grow. Let me fiddle with it a bit.

Reply:
Actually, `24ch` is looking fine to me now, so I'll just change it back and leave it at that.

Reply:
> I had a crisis of faith about my understanding of padding.

😂